Analyzer/plugins/Analyzer.cc

```diff
@@ -12,35 +12,9 @@
 // Modifications by Dylan Angie Frank Apparu
 //                  and Tamas Almos Vami
 
-// v19p0
-// - change double to float
-// - create fillDescription
-// - intro ptErrOverPt vs ptErrOverPt2
-// - change the order of preselection cuts
-// - N-1 plots
-// - Add two more cutflow histos, change boundary for ptErrOverPt2
-// - Fix logic for new cutflow, fix the  change boundary for ptErrOverPt2
-// - Make cuts into an array
-// - Fix logic with not used variales
-// - Change the cut flow order
-// - Add Ih vs Is plot in preselection, change boundary for dxy/dz plots
-// - Change dxy/dz cut default
-// - Add plots for MiniIsol, MET, mT
-// - Change MiniIsol definition, and plot range, move it to preselection
-// - Change EoP to 0.8, then to 2.0 (essentially no cut)
-// - Change to allTrackMCMatch
-// - 18p3: PF matching to gentracks, change the binning of MiniIso histo
-// - 18p4: fix for cutflowProbQfirst index, get rid of EoP cut
-// - 18p5 change to new templates
-// - 18p5: remove TK iso
-// - 18p8: Add postPreselection plots
-// - 19p0: One try with TOF
-// - 19p1: Change mass binning, remove massT cut
-// - 19p3: Simplify probQ cut, change mini-iso def
-// - 19p4: Change mini-iso binning
-// - 19p5: use charged iso in cutflow, dont cut away out of bound probs, only in preselection
-// - 19p6: intro CutFlowEta and PerGenID
-// - 19p7: intro NumEvents and HSCPCandidateType, for comparrison, put back EoP cut and TkIso cut (will remove in 19p8)
+//v20
+// - v20.1 add Regions used to vlaidate the background estimate method
```

Comment:
We need to figure out a better versioning system when we both develop... I'm actually at 21.0 now (it's my fault, I didnt PR my changes)

Reply:
What's your proposal?

Reply:
Version by names maybe? v20p1_Dylan? while I have v20p9_Tamas and then we somehow call the sum of the two v21 in the common code? Not sure....